proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -83,9 +83,6 @@ message PluginConfig {
   // Parameters that should be provided to the plugin. These are
   // joined with a "," before being provided to the plugin at runtime.
   repeated string parameters = 3;
-  // True if the source plugin has been deleted. This dooms
-  // the template.
-  bool deleted = 4 [deprecated=true];
```

Comment:
As a best practice, we should either use `reserved` or decrement the field numbers. Leaving the `4` omitted will cause confusing to the next person.

Reply:
@cyinma When you get a chance, can you update the Buf style guide ?

Framing the question: "How do I remove a field from our public Protobuf definitions?"